tensorflow_quantum/core/ops/cirq_ops_test.py

```diff
@@ -423,6 +421,45 @@ def test_sample_empty_circuit(self):
             cirq.Circuit()).SerializeToString()
         _ = test_op([test_empty_circuit], [], [[]], [10])
 
+    def test_get_cirq_samples_general(self):
+        """Tests that a general cirq.Sampler is compatible with sampling."""
+
+        class DummySampler(cirq.Sampler):
+            """Mock general cirq.Sampler."""
+
+            def run_sweep(self, program, params, repetitions):
```

Comment:
With the changes to `batch_util`, I haven't been able to get this test to pass.  I get length 5 samples (i.e. `[1, 1, 1, 1, 1]`) for every circuit in the batch, even though most circuits have fewer qubits.  Not sure what's going wrong.  Could this be something wrong about how this gets fed to the Pool in `batch_util`?

Reply:
I think it is due to the type casting issue. Please refer to the `_sampler_worker_func()` that I mentioned. Here, you can see that the output results are constructed as `bool`. which means, the value `254` is recognized as `1`, not `-2`?